ccm_web/client/src/components/GradebookUploadConfirmationTable.tsx

```diff
@@ -5,6 +5,7 @@ interface StudentGrade {
   rowNumber: number
   uniqname: string
   grade: string
+  overrideGrade: string | undefined

```

Comment:
The header name first letter should be capitalized.

Reply:
@pushyamig  Do you mean in the table header?

Reply:
Yes

Reply:
Ok I changed that.  This was not where that comes from.